packages/entity/src/EntityCompanionProvider.ts

```diff
@@ -226,6 +226,13 @@ export default class EntityCompanionProvider {
     });
   }
 
+  getQueryContextProviderForDatabaseAdaptorFlavor(
```

Comment:
Might want to make `getTableDataCoordinatorForEntity` call this so that the behavior between the two methods stays in sync.